guides/source/getting_started.md

```diff
@@ -1659,7 +1659,7 @@ You can also update the Edit and Destroy links on the
 <%= link_to "Back", products_path %>
 <% if authenticated? %>
   <%= link_to "Edit", edit_product_path(@product) %>
-  <%= button_to "Destroy", @product, method: :delete, data: { turbo_confirm: "Are you sure?" } %>
+  <%= button_to "Delete", @product, method: :delete, data: { turbo_confirm: "Are you sure?" } %>
```

Comment:
Can you also update `Destroy` to `Delete` in https://github.com/rails/rails/blob/92b3160dd65f31143fe238b27632bafddbfd1d62/guides/source/getting_started.md#L1653